vars/common-vars.yml

```diff
@@ -47,21 +47,21 @@ ucp_namespace_name: "{{ ucp_namespace | default('ucp') }}"
 
 # Variables used for building airship images locally
 suse_distro_identifier: opensuse_15
-suse_airship_components_base_image: "opensuse/leap:15.0"
+suse_airship_components_base_image: "opensuse/leap:15.1"
 
-# check if argument is set in commandline for airship_local_images_flag value
+# Flag to be used when just want to use previously built local airship component images
+# without need of re-building them.
+suse_airship_use_local_images: "{{ lookup('env','AIRSHIP_USE_LOCAL_IMAGES') | default('False', true) | bool }}"
+
+# check if argument is set in commandline for suse_airship_local_images_flag value
 # otherwise use corresponding environment variable to determine it
-suse_airship_build_local_images: "{%- if airship_local_images_flag is defined -%}
-                                    {{ airship_local_images_flag | bool }}
+suse_airship_build_local_images: "{%- if suse_airship_use_local_images_flag is defined -%}
```

Comment:
Why we are tying build local images flag to use local images flag. With this change, if use local images is enabled, then we are going to build local images everytime which is not a desired and unnecessary behavior. 
What's the reason for changing the logic now ?

Reply:
> Why we are tying build local images flag to use local images flag. With this change, if use local images is enabled, then we are going to build local images everytime which is not a desired and unnecessary behavior.
> What's the reason for changing the logic now ?

I tested it locally and it seems to be pulling image if export AIRSHIP_USE_LOCAL_IMAGES=true. 
so it not building image everytime.

Reply:
Looks like for building image both flags needs to be set. So reverting those changes.